.github/actions/junit2jira/action.yaml

```
@@ -0,0 +1,46 @@
+name: 'junit2jira'
+description: 'Convert junit failures into jira tickets'
```

```suggestion
description: 'Convert test failures into Jira tickets'
```
Same rationale as below.